source/manual/data-gov-uk-supporting-ckan.html.md

```diff
@@ -8,43 +8,55 @@ last_reviewed_on: 2018-05-21
 review_in: 3 months
 ---
 [ckan]: https://ckan.org
-[ckanext-datagovuk]: apps/ckanext_datagovuk
-[ckan deployment]: https://github.com/alphagov/datagovuk_ckan_deployment
-[infrastructure]: https://github.com/alphagov/datagovuk_infrastructure
+[ckanext-datagovuk]: /apps/ckanext-datagovuk.html
 [ckandocs]: http://docs.ckan.org/en/ckan-2.7.3/maintaining/paster.html
 
 ## Environments
 
-We currently have one environment: development.
+There are three currently three environments for [CKAN]:
 
-Work is currently underway to define the integration, staging and production environments for [CKAN][ckan]. You can see
-how [CKAN][ckan] fits into data.gov.uk in the [infrastructure] documentation.
+- [Live](https://data.gov.uk) — co-prod3.dh.bytemark.co.uk
+- [Test](https://test.data.gov.uk) — co-prod2.dh.bytemark.co.uk
+- Development — co-dev1.dh.bytemark.co.uk
+
+We are in the process of migrating [CKAN] to standard GOV.UK infrastructure.
 
 ## Applications
 
-[ckanext-datagovuk] is the primary CKAN extension that is deployed as part of the [CKAN deployment][ckan deployment].
-Although other extensions are used in that deployment, [ckanext-datagovuk] is the one that contains our changes
-to functionality and styling.
+[ckanext-datagovuk] is the primary [CKAN] extension. Although other extensions are used in the deployment, 
+[ckanext-datagovuk] is the one that contains our changes to functionality and styling.
```

Comment:
On legacy DGU, the main extension is http://github.com/datagovuk/ckanext-dgu, rather than ckanext-datagovuk (which is the main extension on the new CKAN).

Reply:
Fixed up and rebased.